symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -15,14 +16,41 @@ protected function configureContainer(ContainerConfigurator $container): void
     {
         $container->import('../config/{packages}/*.yaml');
         $container->import('../config/{packages}/'.$this->environment.'/*.yaml');
-        $container->import('../config/{services}.yaml');
```

Comment:
I'm not convinced removing the base `services` and `routes` files is a great step forward. I think the less we have to tweak the Kernel in userland, the best it is. Especially I don't find this much friendly for newcomers.

Reply:
The Kernel holds nothing but configuration. The Kernel *is* configuration. Look at it.
I get this is surprising, like anything new and unexpected.
Yet the Kernel is the perfect place to do configuration - that's its only purpose actually.

Like I mentioned to @yceruto, the Kernel is actually much better than a random file in a folder, because *it has context*: the `use`, the type hint of the method, the MicroKernelTrait.

All these are the heart of a better configuration system than work with autosuggestions.

Reply:
Newcomers will be able to discover the framework using autocompletion. I call that more friendly to them :) Also using PHP can be seen as more friendly.
The DI configuration for an app is very thin nowadays, there is no need to build a full architecture to organize it - at least to at the beginning of a project.
(note that I'm specifically talking about the app's config. vendors do need dedicated files, yaml is the best for them)